finetune/encoding/sequence_encoder.py

```diff
@@ -139,6 +141,8 @@ def finetune_to_indico_sequence(
                         # and only separated by whitespace
                         and item["end"] <= raw_annotation_end
                         and not raw_text[item["end"]: raw_annotation_start].strip()
+                        # and not BIO tagging
+                        and not bio_tagging
```

Comment:
This doesn't seem like a great way to get this to work (possible edge cases that break it? Can't come up with any now), but I'm not sure how else to do it. As we currently merge spans and strip the BIO prefixes in the original `predict()` call before converting, `finetune_to_indico` sees sequential spans of the same tag type and tries to merge them.

I tried doing the merging / prefix stripping here instead, but then we run into the issue of `predict()` merging  consecutive `B-` spans. If we merge spans in `predict()` and remove prefixes in `finetune_to_indico`, we have the issue of the initial `B-` tag breaking when sub-token prediction isn't on.